js/index.js

```diff
@@ -26,24 +26,62 @@ const cards = [
 ];
 
 const memoryGame = new MemoryGame(cards);
+console.log(memoryGame);
+memoryGame.shuffleCards();
 
 window.addEventListener('load', event => {
   let html = '';
   memoryGame.cards.forEach(pic => {
+    //console.log(`${pic.name}`);
     html += `<div class="card" data-card-name="${pic.name}">`;
-    html += `<div class="back" name="${pic.img}"></div>`;
+    /* console.log(pic.name); */
+    html += `<div class="back" name="${pic.name}"></div>`;
     html += `<div class="front" style="background: url(img/${pic.img}) no-repeat"></div>`;
     html += `</div>`;
+
   });
 
+  function toggle (cardElement, classList){
+classList.forEach(name => cardElement.classList.toggle(name))
+  }
+
   // Add all the divs to the HTML
   document.querySelector('#memory-board').innerHTML = html;
-
+  
   // Bind the click event of each element to a function
-  document.querySelectorAll('.card').forEach(card => {
-    card.addEventListener('click', () => {
-      // TODO: write some code here
-      console.log(`Card clicked: ${card}`);
+  document.querySelectorAll(".card").forEach(card => {
+    card.addEventListener("click", () => {
+      let cardName = card.getAttribute("data-card-name");
+      console.log(`card picked is ${cardName}`);
+      card.querySelectorAll(".card div").forEach(div =>{
+        div.classList.toggle("front")
+        div.classList.toggle("back")
+      })
+      //console.log(`bla`,memoryGame.pickedCards)
+      memoryGame.pickedCards.push(cardName)
+      //console.log(`nlanla`,memoryGame.pickedCards)
+      let card1 = memoryGame.pickedCards[0];
```

Comment:
Will card1 be reassigned? Is "let" the best keyword for this variable?
Same question for variable card2.